lms/djangoapps/courseware/models.py

```diff
@@ -248,3 +254,95 @@ class Meta(object):   # pylint: disable=missing-docstring
 
     field = models.CharField(max_length=255)
     value = models.TextField(default='null')
+
+
+SCORE_CHANGED = Signal(
+    providing_args=[
+        'points_possible',  # Maximum score available for the exercise
+        'points_earned',   # Score obtained by the user
+        'user',  # Integer User ID
+        'course',  # Unicode string representing the course
+        'usage'  # Unicode string indicating the courseware instance
```

Comment:
Could you please call these `course_id` and `usage_id`? We're not terribly consistent internally, but the most common convention I've seen is to use `course` for an actual course module, `course_id` for the string representation of the ID, and `course_key`for the CourseKey/Locator.